exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporterBuilder.java

```diff
@@ -53,6 +56,21 @@ public ZipkinSpanExporterBuilder setEncoder(BytesEncoder<Span> encoder) {
     return this;
   }
 
+  /**
+   * Sets the Function that is responsible for transforming an OpenTelemetry {@link SpanData} into
+   * an instance of a Zipkin {@link Span}.
```

Comment:
probably worth providing a pointer to the default implementation and documenting that it is the default.